bridgesample/src/androidTest/java/com/livefront/bridgesample/scenario/util/CurrentActivity.kt

```diff
@@ -6,17 +6,19 @@ import androidx.test.runner.lifecycle.ActivityLifecycleMonitorRegistry
 import androidx.test.runner.lifecycle.Stage.RESUMED
 
 /**
- * @return the current activity being displayed
+ * @return the current activity being displayed.
  */
-fun getCurrentActivity(): Activity? {
+@Throws(NoSuchElementException::class)
```

Comment:
If we don't actually expect this to happen we should probably just leave this part off.

Reply:
Done.